app/models.py

```diff
@@ -0,0 +1,8 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Actor:
+    id: int
+    first_name: str
+    last_name: str
```

Comment:
The field `last_name` in the `Actor` dataclass does not match the column name `second_name` used in the `ActorManager` class. To maintain consistency, consider renaming this field to `second_name` or updating the database operations to use `last_name`.